### ISSUE TITLE
[Please read the message below]

### ISSUE THREAD

**Issue by sh1man** [solidjs/solid-start#1306]
### Please do not submit this issue.

- [X] I understand

**Comment by ryansolid** [comment]
I agree. While I don't think it belongs as a template in this repo I've been thinking it would be cool to create an updated version of the Realworld Demo App (https://github.com/solidjs/solid-realworld)

**Comment by sh1man999** [comment]
yes, I'm really looking forward to the updated version, it's very interesting for beginners

**Comment by sh1man999** [comment]
i would like to see the use of trcp in this project
